File: src/domain/mappers.ts
import { SafeUser, StoredUser } from './models.js';

export function toSafeUser(user: StoredUser): SafeUser {
  return {
    id: user.id,
    email: user.email,
    displayName: user.displayName,
    createdAt: user.createdAt,
  };
}

File: src/domain/mappers.py
from src.domain.models import SafeUser, StoredUser


def to_safe_user(user: StoredUser) -> SafeUser:
    return SafeUser(
        id=user.id,
        email=user.email,
        display_name=user.display_name,
        created_at=user.created_at,
    )
